scripts/semantic-commits/get-binary-release-data.js

```diff
@@ -86,7 +86,7 @@ const getReleaseData = async (latestReleaseInfo) => {
       pull_number: ref.issue,
     })
 
-    const associatedIssues = getLinkedIssues(pullRequest.body)
+    const associatedIssues = pullRequest.body ? getLinkedIssues(pullRequest.body) : []
```

Comment:
If the PR body is empty, then there aren't any associated issues. Don't throw an error